docs/nidcpower/rep_caps.rst

```diff
@@ -42,9 +50,14 @@ instruments
 
         .. code:: python
 
-            session.instruments['0-2'].channel_enabled = True
+            print(session.instruments['Dev1'].serial_number)
+
+        prints :py:attr:`serial_number` for instruments 'Dev1'.
+
+        .. code:: python
 
-        passes a string of :python:`'0, 1, 2'` to the set attribute function.
+            print(session.instruments['Dev1', 'Dev2', '3rdDevice'].serial_number)
```

Comment:
>I found 3rdDevice confusing.

I just didn't want users to think it had to start with "Dev". I have the same concern with the examples you gave.
Maybe my blurb about the basic element should specify "device alias string" for instruments?

Reply:
`PXI1Slot2` is a good alternative.

Reply:
Right, just use something that matches defaults. This will be informative enough and not confused anyone.

> Maybe my blurb about the basic element should specify "device alias string" for instruments?

I question how familiar customers are with term "device alias". It's certainly not in our APIs. Let's avoid,